languages/elixir/config.json

```diff
@@ -220,6 +220,19 @@
         "uuid": "83c31f5f-fc94-4315-9053-df5a5f5d5fb7",
         "concepts": ["io"],
         "prerequisites": ["strings", "maps", "atoms"]
+      },
+      {
+        "slug": "captains-log",
```

Comment:
Maybe this one needs the name key, since I'm not sure the system will be able to infer the `'` for `Captain's Log`

Reply:
Yes, definitely, good catch!